developer-experience/readme.md

```diff
@@ -0,0 +1,130 @@
+
+## Participation
+
+The entire development team should take responsibility in ensuring positive
+
+## Facilitation Guidance
+
+The following outline several strategies that can be adopted to promote a positive developer experience.
+
+### Establish Hotkeys
+
+Assign hotkeys to each of the essential tasks.
+
+[`TODO: Suggest hotkeys for mac`]
+
+| Task                 | Windows      |
+| -------------------- | ------------ |
+| Build                | CTRL+SHIFT+B |
+| Test                 | CTRL+R,T     |
+| Start With Debugging | F5           |
+
+### The F5 Contract
+
+The F5 contract aims for the ability to run the end-to-end solution with the following steps.
+
+1. Clone - git clone [`my-repo-url-here`]
+2. Configure - set any configuration values that need to be unique to the individual (i.e. update a .env file)
+3. Press F5 - launch the solution with debugging attached.
+
+Most IDE's have some form of a task runner that can be used to automate the build, execute, and attach steps. Try to leverage these such that the steps can all be run with as few manual steps as possible.
+
+### Pay Attention to the Struggles
+
+Ask the team to surface the areas they spend significant time in that they consider non-value-add activities. For example, do they need to deploy their changes to an environment off their laptop before they can validate if what they did worked. Rather than debugging locally, do they have to do this repetitively to get to a working solution? Does this take several minutes each iteration? Does this block other developers due to the contention on the environment?
+
+The following are a few strategies for surfacing dev experience struggles
+
+- Retrospectives. Create a card that describes the struggle
+- Standup Blockers. Call out being blocked not being able to test change and why that cant be done.
+
+#### Fixing the Struggles
+
+Once raised, use the following steps to capture and create **actionable** fix.
+
+1. Define the problem - End to end testing is only possible when the solution is deployed to environment in Azure
+2. Identify a potential solution - Ability to run automated end-to-end tests on the developer machine.
+3. Justify fixing the problem - The team has identified this issue as the root cause for 3 stories not completing within their sprint over the past 2 sprints.
+
+### Single IDE Instance
+
+Avoid requiring multiple instances of the IDE to running concurrently to run and debug the solution locally. This will negatively impact the above measure. Additionally, that negative impact is multiplied by each instance required.
+
+- Configuration steps * IDE instances
+- Build steps * IDE instances
+- Start/Debug steps * IDE instances
+- Stop steps * IDE instances
+- Run test steps * IDE instances
+- Documenting all of the above * IDE instances
+
+### Single Repository

```

Comment:
I think the real problem is having a lot of repositories *without a way to effectively manage build artifacts and versioned dependencies.* 

I like the phrasing you have in the description below, as it focuses on keeping a single *solution* in one repository. That's key, but it does not match your title for this section. Perhaps a re-wording could be *Minimizing number of repositories*